odxtools/diagnostictroublecode.py

```diff
@@ -18,7 +20,7 @@ class DiagnosticTroubleCode:
     short_name: Optional[str]
     text: Optional[str]
     display_trouble_code: Optional[str]
-    level: Union[bytes, None]
+    level: Union[int, None]
```

Comment:
Are you sure about this change?

Reply:
quite: the XSD says its an `xsd:unsignedByte`, which -- if I understand this correctly -- just an integer in the 0-255 range...